### ISSUE TITLE
Untranslatable strings in Pages.cpp, RLibrary.cpp, can't simple fix by tr()

### ISSUE THREAD

**Issue by MaksVasilev** [GoldenCheetah/GoldenCheetah#3513]
I try use tr() in "static const SimBicyclePartEntry arr[] = {}" (on 2225 line), but have conversion string type error:

```
Gui\Pages.cpp(2251): error C2440: 'initializing': cannot convert from 'QString' to 'const char *'
Gui\Pages.cpp(2251): note: No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called
```

Commit with error:
https://github.com/MaksVasilev/GoldenCheetah/commit/1b26b223b64bc759a56f1b6eac1cc73ac5a58793

**Comment by amtriathlon** [comment]
The problem is the code added by https://github.com/GoldenCheetah/GoldenCheetah/pull/3332 doesn’t comply with the first requirement to enable translations (Using QString for All User-Visible Text): https://doc.qt.io/qt-5/i18n-source-translation.html

**Comment by MaksVasilev** [comment]
Addition:

Untranslatable strings in RLibrary.cpp like as:
`        errors << "R_HOME has not been configured in options or the system environment"
                  " so we looked in the common places to find the R install.\n\n";
`

In all operation with QString "errors".

**Comment by MaksVasilev** [comment]
Addition:

Untranslatable strings in **Charts/Overview.cpp** like as:

`            add = new MetaOverviewItem(space, "Sport", "Sport");
            space->addItem(2,0,5, add);

            add = new MetaOverviewItem(space, "Workout Code", "Workout Code");
            space->addItem(3,0,5, add);

            add = new MetricOverviewItem(space, "Duration", "workout_time");
            space->addItem(4,0,9, add);
`

**Comment by amtriathlon** [comment]
> Addition:
> 
> Untranslatable strings in RLibrary.cpp like as:
> ` errors << "R_HOME has not been configured in options or the system environment" " so we looked in the common places to find the R install.\n\n";`
> 
> In all operation with QString "errors".

This is explained in the same document under "Translating Non-Qt Classes", we have used that solution in other places to enable tr() macro:
```
class MyClass
{
    Q_DECLARE_TR_FUNCTIONS(MyClass)

public:
    MyClass();
    ...
};
```

**Comment by amtriathlon** [comment]
> Addition:
> 
> Untranslatable strings in **Charts/Overview.cpp** like as:
> 
> ```
>             space->addItem(2,0,5, add);
> 
>             add = new MetaOverviewItem(space, "Workout Code", "Workout Code");
>             space->addItem(3,0,5, add);
> 
>             add = new MetricOverviewItem(space, "Duration", "workout_time");
>             space->addItem(4,0,9, add);
> ```

This seems a case were just adding at tr() should work, but only on the name argument, the symbol one should not be translated.

**Comment by MaksVasilev** [comment]
Right now I don't have the ability to compile to check (covid isolation). I only have linquist without compiler and I translate the current code.

**Comment by amtriathlon** [comment]
If you are on Windows, you can enable AppVeyor on you own fork or generate a PR, in both cases the build result will be available for download at the Artifacts tab.

**Comment by MaksVasilev** [comment]
Sorry, I'm not coder. I'm a coach, physiologist and translator.

Addition:

```Metrics/PDModel.h:        QString name()   { return "Classic  Parameter"; } ```

UserChart.cpp:

```    tabs->addTab(axisWidget, "Axes");
...
    tabs->addTab(cs, "Chart");
...
    tabs->addTab(seriesWidget, "Series");```

**Comment by amtriathlon** [comment]
No problems, thanks for your translation!
Lets keep this open to be addressed when v3.6 be more complete.

**Comment by ericchristoffersen** [comment]
Im happy to run all user visible strings in bicycle sim page through tr. Have something soon, sorry.
